2.extract-features/README.md

```diff
@@ -2,9 +2,49 @@
 
 In this module, we present our pipeline for extracting features from the Cell Health data.
 
-### Feature Extraction
+# CellProfiler Feature Extraction
```

Comment:
Consider making this a header-2 (`##`) as there is already a top-level header defined. The general guidance here stems from HTML, where it is recommended to use only header-1 per document ([see here for more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page)).